rhtconsulting/rhc-ose
# Added new docker container for jekyll testing. Moved docker containers into new folder

#### What does this PR do?

Adds a new Docker container for Jekyll testing. Moved new jekyll-asciidoc and existing openstack-client container to new `docker` folder
#### How should this be manually tested?

To test the new jekyll container, have a jekyll project available on the local machine [openshift-playbooks#jekyll-asciidoc-publishing](https://github.com/rhtconsulting/openshift-playbooks/tree/jekyll-asciidoc-publishing)

Inside the `docker/jekyll-asciidoc` folder, execute the following

```
./run.sh -d=<directory_containing_jekyll>
```

New image should be built and container should start, build and serve content.

Browse to the following location:

```
http://<docker_ip>:4000
```
#### Is there a relevant Issue open for this?

None
#### Who would you like to review this?

/cc @etsauer
